includes/admin/onboarding/onboarding.php

```diff
@@ -0,0 +1,637 @@
+		if ( ! empty( $products->posts ) ) {
+			$this->onboarding_steps['products']['step_intro'] = 'Let\'s get started with your next great product.';
+		}
+
+		// Set step index in the array and load ajax handlers.
+		$index = 1;
+		foreach ( $this->onboarding_steps as $key => $value ) {
+			$this->onboarding_steps[ $key ]['step_index'] = $index;
+			$index++;
+
+			// Initialize step logic.
+			if ( function_exists( 'EDD\\Onboarding\\Steps\\' . $value['step_handler'] . '\\initialize' ) ) {
+				call_user_func( 'EDD\\Onboarding\\Steps\\' . $value['step_handler'] . '\\initialize' );
+			}
+		}
+	}
+
+	/**
+	 * Set current onboarding step.
+	 *
+	 * @since 3.2
+	 */
+	public function set_current_onboarding_step() {
+		// If Onboarding hasn't started yet, we force the first default step.
+		if ( ! $this->onboarding_started ) {
+			return;
+		}
+
+		// User is requesting a specific step.
+		if( isset( $_GET['current_step'] ) ) {
+			$this->current_step = sanitize_key( $_GET['current_step'] );
+		} else {
+			$this->current_step = sanitize_key( get_option( 'edd_onboarding_latest_step', $this->current_step ) );
+		}
+
+		// If requested step does not exist, abort.
+		if ( ! isset( $this->onboarding_steps[ $this->current_step ] ) ) {
+			wp_die( __( 'Unknown Onboarding Step.', 'easy-digital-downloads' ), __( 'Onboarding Wizard', 'easy-digital-downloads' ), 404 );
+		}
+
+		$this->current_step_index = $this->onboarding_steps[ $this->current_step ]['step_index'];
+		update_option( 'edd_onboarding_latest_step', $this->current_step, false );
+	}
+
+	/**
+	 * Get previous step.
+	 *
+	 * @since 3.2
+	 */
+	public function get_previous_step() {
+		$internal_step = $this->current_step_index - 2;
+		$step_keys     = array_keys( $this->onboarding_steps );
+		if ( isset( $step_keys[ $internal_step ] ) ) {
+			return $step_keys[ $internal_step ];
+		}
+
+		return false;
+	}
+
+	/**
+	 * Get current step.
+	 *
+	 * @since 3.2
+	 */
+	public function get_current_step() {
+		return $this->current_step;
+	}
+
+	/**
+	 * Get current step details.
+	 *
+	 * @since 3.2
+	 */
+	public function get_current_step_details() {
+		return $this->onboarding_steps[ $this->get_current_step() ];
+	}
+
+	/**
+	 * Get next step.
+	 *
+	 * @since 3.2
+	 */
+	public function get_next_step() {
+		$internal_step = $this->current_step_index;
+		$step_keys     = array_keys( $this->onboarding_steps );
+		if ( isset( $step_keys[ $internal_step ] ) ) {
+			return $step_keys[ $internal_step ];
+		}
+
+		return false;
+	}
+
+	/**
+	 * Get pagination.
+	 *
+	 * @since 3.2
+	 */
+	public function get_step_pagination() {
+		return array(
+			'previous' => $this->get_previous_step(),
+			'current'  => $this->get_current_step(),
+			'next'     => $this->get_next_step(),
+		);
+	}
+
+	/**
+	 * Onboarding Wizard subpage screen.
+	 *
+	 * @since 3.2
+	 */
+	public function onboarding_wizard_sub_page() {
+		$onboarding_initial_style = ( ! $this->onboarding_started ) ? ' style="display:none;"' : '';
+		?>
```

Comment:
This is going to sound weird but I'm going to ask that everything here be wrapped in a container that just has a class of `wrap`. The reason I'm asking for it is that WP core uses that CSS class to manage some margins so that the page content appears centered in its container next to the admin menu. (You could maybe just add the class to your `edd-onboarding` container, and change the CSS rule to add the top margin to be more specific.)